package.json

```diff
@@ -56,9 +57,10 @@
     "jshint": "jshint lib/rest/** lib/base/** lib/http/**",
     "jscs": "eslint lib/base/**/**.js lib/http/**/**.js --fix",
     "check": "npm run jshint && npm run jscs",
-    "ci": "npm run test && npm run nsp",
```

Comment:
I dislike the cluster tests being run on `make test` or `npm test`. These should _only_ be run when we're specifically targeting cluster tests.